modin/experimental/engines/omnisci_on_ray/io.py

```diff
@@ -276,9 +317,6 @@ def read_csv(
                 null_values=None,
                 true_values=None,
                 false_values=None,
-                # timestamp fields should be handled as strings if parse_dates
-                # didn't passed explicitly as an array or a dict
-                timestamp_parsers=[""] if isinstance(parse_dates, bool) else None,
```

Comment:
this trick works only for `timestamp` dtypes, since Arrow 3.0 it [started to also infer](https://issues.apache.org/jira/browse/ARROW-11247) `date` dtypes, whose infer-restriction have to be specified by explicit "string" column's data-type via `column_types` parameter